docs/docs/howto/clusterconnectivity/aiven-kafka-cluster-sasl-ssl-protocol.rst

```diff
@@ -6,7 +6,7 @@ This section provides you with information on how to connect Aiven for Apache Ka
 Prerequisite
 ------------
 
-* Set up the connection between the Klaw APIs (Core API and Cluster API), see :doc:`klaw-core-with-clusterapi`. This involves configuring the ``klaw.clusterapi.url`` setting in the Klaw UI and testing the connectivity to ensure the two APIs can communicate.
+* Set up the connection between the Klaw APIs (Core API and Cluster API) to use secure SSL, see :doc:`klaw-core-with-clusterapi`. This involves configuring the ``klaw.clusterapi.url`` setting in the Klaw UI and testing the connectivity to ensure the two APIs can communicate over https.
```

Comment:
"to use secure SSL" it can also be without ssl right ?

Reply:
My apologies I think you are right, I tested "Connect with Aiven for Apache Kafka® Cluster using SSL protocol" with http and it wouldn't work until i upgraded to SSL but with SASL it might be different I will update that.